memcached-image/memcached-image.xsl

```diff
@@ -0,0 +1,117 @@
+  please check README
+</xsl:comment>
+
+<image schemaversion="6.9" name="memcached-image" xmlns:suse_label_helper="com.suse.label_helper">
+
+  <description type="system">
+    <author>Thomas Bechtold</author>
+    <contact>tbechtold@suse.com</contact>
+    <specification>memcached container</specification>
+  </description>
+
+  <preferences>
+    <type image="docker">
+      <xsl:attribute name="derived_from">
+        <xsl:value-of select="param/image" />
+      </xsl:attribute>
+      <containerconfig
+          tag="latest"
+          user="memcached"
+          additionaltags="%LONG_VERSION%,%LONG_VERSION%-%RELEASE%"
+          maintainer="SUSE Cloud Team &lt;cloud@suse.com&gt;">
+        <xsl:attribute name="name">
+          <xsl:value-of select="param/name" />
+        </xsl:attribute>
+
+        <expose>
+          <port number="11211/tcp"/>
+          <port number="11211/udp"/>
+        </expose>
+        <subcommand execute="/usr/sbin/memcached"/>
+        <entrypoint execute="/usr/local/bin/docker-entrypoint.sh"/>
+
+         <labels>
+          <suse_label_helper:add_prefix>
+            <xsl:attribute name="prefix">
+              <xsl:value-of select="param/prefix" />
+            </xsl:attribute>
+
+            <label name="org.opencontainers.image.title" value="memcached container"/>
+            <label name="org.opencontainers.image.description" value="memcached container"/>
+            <label name="org.opencontainers.image.version" value="%LONG_VERSION%-%RELEASE%"/>
+            <label name="org.opencontainers.image.created" value="%BUILDTIME%"/>
+            <label name="org.opencontainers.image.vendor" value="openSUSE"/>
+            <label name="org.openbuildservice.disturl" value="%DISTURL%"/>
+
+            <label name="org.opensuse.reference">
+              <xsl:attribute name="value">
+                <xsl:value-of select="param/reference"/>
+              </xsl:attribute>
+            </label>
+
+          </suse_label_helper:add_prefix>
+
+          <xsl:for-each select="param/labels/label">
+            <label>
+              <xsl:attribute name="name">
+                <xsl:value-of select="name"/>
+              </xsl:attribute>
+              <xsl:attribute name="value">
+                <xsl:value-of select="value"/>
+              </xsl:attribute>
+            </label>
+          </xsl:for-each>
+
+        </labels>
+
+        <history author="Thomas Bechtold &lt;tbechtold@suse.com&gt;">memcached</history>
+     </containerconfig>
+    </type>
+    <version>1.0.0</version>
+    <packagemanager>zypper</packagemanager>
+    <rpm-check-signatures>false</rpm-check-signatures>
+    <rpm-excludedocs>true</rpm-excludedocs>
+  </preferences>
+
+  <repository type='rpm-md'>
+    <source path='obsrepositories:/'/>
+  </repository>
+
+  <packages type="image">
+    <package name="ca-certificates"/>
+    <package name="memcached"/>
+    <archive name="memcached-docker-entrypoint.tar"/>
```

Comment:
Thanks again for this ! I was looking for a way to add files to the image that are brought in by build source services. This is golden !

Reply:
:)